apps/condo/domains/common/components/HCaptcha.tsx

```diff
@@ -0,0 +1,84 @@
+import HCaptcha from '@hcaptcha/react-hcaptcha'
+import { notification } from 'antd'
+import get from 'lodash/get'
+import getConfig from 'next/config'
+import React, { createContext, FC, useCallback, useContext, useMemo, useRef } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Button } from '@open-condo/ui'
+
+
+type IHCaptchaContext = {
+    executeCaptcha: () => Promise<string>
+}
+
+const HCaptchaContext = createContext<IHCaptchaContext>({
+    executeCaptcha: async () => '',
+})
+
+const useHCaptcha = (): IHCaptchaContext => useContext(HCaptchaContext)
+
+const HCaptchaProvider: FC = ({ children }) => {
+    const intl = useIntl()
+    const requestFailedMessage = intl.formatMessage({ id: 'global.errors.requestFailed.title' })
+    const OkMessage = intl.formatMessage({ id: 'OK' })
+
+    const { publicRuntimeConfig: { hCaptcha: hCaptchaConfig, disableCaptcha } } = getConfig()
```

Comment:
is `disableCaptcha` are equal than no `hCaptchaConfig`?

Reply:
No. To turn off the captcha, you need to use the `DISABLE_CAPTCHA` flag

Reply:
Now on the front-end the captcha can be without configuration or disabled. If this is so, then instead of the token the text condo-captcha-disabled will be sent (this is just a placeholder, there could be any other text here, it does not affect anything, just the token field is required in the API)

Everything on the server remains the same. If the DISABLE_CAPTCHA flag is set, then the captcha check will not work. If for some reason you have not specified the configuration, the corresponding warnings and errors will be displayed. Therefore, if you want to disable the captcha, then this must be done explicitly through the DISABLE_CAPTCHA flag